src/librbd/operation/TrimRequest.cc

```diff
@@ -170,11 +180,11 @@ bool TrimRequest<I>::should_complete(int r)
 
 template <typename I>
 void TrimRequest<I>::send() {
-  send_copyup_objects();
+  send_update_objectmap();
 }
 
 template <typename I>
-void TrimRequest<I>::send_copyup_objects() {
+void TrimRequest<I>::send_update_objectmap() {
```

Comment:
Minor: this state doesn't seem to do anything besides pick which method to invoke next -- why not just combine it w/ ```send_pre_copyup``` and skip to ```send_pre_remove``` if copyup isn't needed.